javascript/javascript/JavaScriptParser.g4

```diff
@@ -386,6 +399,12 @@ singleExpression
     | '(' expressionSequence ')'                                            # ParenthesizedExpression
     ;
 
+initializer
+// TODO: must be `= AssignmentExpression` and we have such label alredy but it doesn't respect the specification.
+//  See https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-Initializer
+    : '=' singleExpression
```

Comment:
I haven't implemented `AssignmentExpression` rule because it requires too many changes. We should do changes step-by-step IMHO.